_posts/2024-03-11-transportation-round-up.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: blog_post
+title: "The Compiler Team's Transportation Round-up"
+subtitle: "Our favorite transit stories from the past year"
+description: "Our favorite transit stories from the past year"
+author: Laney Mangan
+excerpt: " "
+date:   2024-03-11 19:03:13 +0200
+categories: compiler
+---
+---
+I won’t speak for all of my Compiler colleagues, but I will say that 100% of us think that public transportation
+is important. Shoot, did I just speak for all of my colleagues? Oh well. Public transportation is a vital community service;
+it’s good for our communities and our earth. It’s one of the most convenient, affordable, and efficient ways of shuffling
+people through their lives. So, like I said, we think public transportation is important; but we also know it to be occasionally
+frustrating, sometimes surprising, often exciting, and at times, an all-around adventure. So, we took some time to reflect on our
+most notable transit adventures from the past year. Read on to see what the Compiler team has to say about their recent travels.
+
+**Jess**
+
+I took my two COVID babies on public transit for the first time! We took a round-trip journey on the San Diego trolley to
+Old Town and enjoyed a really amazing sunset over Mission Bay on the way home. They thought it was the coolest "choo choo" in the world.
+
+
+**Machiko**
+
+I love taking the Hollywood Bowl Park & Ride bus - I took it exactly four times last year during the summer Bowl season.
+The ticket buying process is seamless -- you buy a ticket online as you are buying your Bowl tickets. There are 14 Park & Ride
+bus stops all over L.A. County, from Torrance, Chatsworth, Pasadena and more. On your concert date at the shuttle time, you show
+your e-ticket to the friendly attendants at the bus stop and get in the bus. The staff at the Bowl always provide services for
+those with disabilities, and they make sure you find your bus on the way home. I would definitely go to more concerts and
+venues if they had the same kind of Park & Ride shuttle service like the Bowl's.
+
+**Olivia**
+
+BART (Bay Area Rapid Transit) officially had all service running on new train cars, retiring all old train cars.
+The new train cars are so clean and much quieter than the old cars, making for a much more enjoyable rider experience.
+They started phasing in new cars in 2018, and the primary line I ride has had a mix of old/new cars in service over
+the last few years, so I never quite knew if I would be riding an old or new car until it showed up. I was very excited
+when the last new car was put into service. And it looks like BART managed to complete this project well under budget! Kudos to them!
+
+**Angela**
+
+I took the Staten Island Ferry, for FREE, in NYC. It was an awesome way to see the skyline, bridges and the Statue of Liberty.
+
+**Shelby**
+
+
+*[Remember when we said public transportation can sometimes be frustrating–yeah. Transit fails are also worth reminiscing on]*
```

Comment:
Let's remove the brackets and add italics to the sentence. It would be good for this sentence to be its own paragraph. It's not super critical; I think the space highlights your internal thoughts.